README.md

```diff
@@ -69,8 +69,10 @@ Workmanager.registerOneOffTask(
 // Periodic task registration
 Workmanager.registerPeriodicTask(
     "2", 
-    "simplePeriodicTask", 
-    frequency: Duration(hours: 1), //When no frequency is provided the default 15 minutes is set.
+    "simplePeriodicTask", \
```

Comment:
Maybe also duplicate this document in the `Workmanager.registerPeriodicTask` documentation